runbot_gitlab_minimal/models/runbot_repo.py

```diff
@@ -113,6 +116,13 @@ def github(self, cr, uid, ids, url, payload=None, ignore_errors=False,
                         json = json[0]
                         json['head'] = {'ref': json['target_branch']}
                         json['base'] = {'ref': json['source_branch']}
+                    if '/commits/' in url:
+                        url = _get_url('/users?search=%s' %
+                                       json['committer_email'], repo.base)
```

Comment:
You are using 2 emails for 1 user
I'm talking about 2 users with 1 same email configured

I means,
User: JesusZapata
Email: jesus@vauxoo.com

User: JesusZapata2
Email: jesus@vauxoo.com